lib/driver/runtimeTool.cpp

```diff
@@ -22,6 +22,21 @@
 #include <vector>
 
 namespace WasmEdge {
+std::ostream &operator<<(std::ostream &OS, uint128_t Value) {
+  std::string S;
+  if (Value < (uint64_t)-1) {
```

Comment:
Use the `std::numeric_limits<uint64_t>::max`

Reply:
`<=`, not `<`

Reply:
fixed